Provis.WebApi/Policy/WorkspaceRolesAuthorizationHandler.cs

```diff
@@ -47,7 +47,7 @@ protected override Task HandleRequirementAsync(AuthorizationHandlerContext conte
                     bodyStr = reader.ReadToEnd();
 
                     JObject jObj = JObject.Parse(bodyStr);
-                    workspaceId = (int)jObj["workspaceId"];
+                    workspaceId = (int?)jObj["workspaceId"];
```

Comment:
Can we get null pointer exception here ?

Reply:
Actually, with proper body in POST/PUT request, we can't get it